main.js

```diff
@@ -0,0 +1,7 @@
+var header = document.querySelector("#scheduled-day");
+
+var days = document.querySelectorAll(".day").forEach(day =>{
+    day.addEventListener('click', function(){
+        header.innerText = "Scheduled Events for " + day.innerHTML;
+    })
+})
```

Comment:
Assigning the return value of the forEach method does not do anything.  forEach returns undefined.

Use event delegation to apply your event listener.

If you did it correctly, you will only need to add one event listener.